etc/profile-a-l/deadlink.profile

```diff
@@ -0,0 +1,70 @@
+# Firejail profile for deadlink
+# Description: Checks and fixes URLs in code and documentation
+# This file is overwritten after every install/update
+# Persistent local customizations
+include deadlink.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist /usr/libexec
+blacklist ${RUNUSER}
+
+noblacklist ${HOME}/.config/deadlink
+
+# Allow python (blacklisted by disable-interpreters.inc)
+#include allow-python2.inc
+include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/deadlink
+whitelist ${HOME}/.config/deadlink
+whitelist ${DOWNLOADS}
+include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
+machine-id
+netfilter
+nodvd
+nogroups
+noinput
+nonewprivs
+noprinters
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,inet
+seccomp
+seccomp.block-secondary
+tracelog
+x11 none
+
+disable-mnt
+private-bin deadlink,python*
+private-cache
+private-dev
+private-etc @network,@tls-ca
+private-tmp
+
+dbus-user none
+dbus-system none
+
+memory-deny-write-execute
+restrict-namespaces
+read-only ${HOME}
+read-write ${HOME}/.config/deadlink
```

Comment:
```suggestion
```

It says "fix URLs", so doesn't it modify the files?

Reply:
It sure can. Fixed!